docs/release-notes/known-issues.md

```diff
@@ -5,6 +5,8 @@ navigation_title: "Known issues"
 
 # Elastic APM Go Agent known issues [elastic-apm-go-agent-known-issues]
 
+Known issues are significant defects or limitations that may impact your implementation. These issues are actively being worked on and will be addressed in a future release. Review the Elastic APM Go Agent known issues to help you make informed decisions, such as upgrading to a new version.
```

Comment:
Should we mention #1719 (see here https://github.com/elastic/sdh-apm/issues/1713 for more context) ? 

It's highly likely we have a memory corruption that manifests when configuration is **NOT** `CaptureBodyOff` so we should probably try to steer people away from triggering it.